apps/codeimage/src/components/PropertyEditor/FrameStyleForm.tsx

```diff
@@ -27,16 +39,24 @@ export const FrameStyleForm: ParentComponent = () => {
           <SuspenseEditorItem
             fallback={<SkeletonLine width={'100%'} height={'26px'} />}
           >
-            <SegmentedField
-              adapt
+            <Select
+              {...paddingOptions.props()}
+              {...paddingOptions.controlled(
+                () => String(frame.store.padding),
+                padding => {
+                  if (typeof padding !== 'undefined') {
+                    frame.setPadding(Number(padding));
```

Comment:
i'd prefer to do the opposite condition:

```ts
if (typeof padding === 'undefined') { 
  rerturn;
}
```

Reply:
Done